releasenotes/notes/add-2q-fractional-gates-to-unitarysynthesis-pass-f66eee29903f5639.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features_synthesis:
+  - |
+    Add a :class:`.TwoQubitControlledUDecomposer` that decomposes any two-qubit unitary
+    in terms of basis two-qubit fractional gates, such as :class:`.RZZGate`
+    (or two-gates gates which are locally equivalent to :class:`.RZZGate` up to single qubit gates).
```

Comment:
I think we need to show an example here of how the class is used from python.

Reply:
done in [55556f8](https://github.com/Qiskit/qiskit/pull/13568/commits/55556f8f4762d960391311a76d2ecc4ee2d3222c)